src/hammer-vlsi/defaults.yml

```diff
@@ -449,6 +449,29 @@ vlsi.inputs:
     #   Requires the semi_auto pin generation mode.
     assignments: []
 
+  # SVG visualization of placement constraints and bumps for the current hierarchical module.
```

Comment:
No. This would require a fundamental rewrite of hierarchical Hammer. When you call `get_placement_constraints`, Hammer goes through the graph and picks out only the constraints that apply to the current node.

Reply:
I think it would be cool to do both but we can just leave it as an issue.